tensorlayer/models/transformer/feedforward_layer.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implementation of fully connected network."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import tensorlayer as tl
+
+
+class FeedForwardLayer(tl.layers.Layer):
```

Comment:
How is this FeedForwardLayer different from conventioinal feedforward layer? If this is a special implementation, can it have a more specific name?

Reply:
FeedforwardLayer here is specifically designed in Transformer model, which includes two transformations. 
The conventional feedforward layer only contains one.
Any suggestion on how the naming?

Reply:
TransformerFeedFordwardLayer?